crates/tracing-instrumentation/src/lib.rs

```diff
@@ -279,7 +279,10 @@ pub fn init_tracing_and_logging(
             .clone()
             .expect("falls back to default value")
         {
-            BindAddress::Uds(p) => console_subscriber::ServerAddr::Unix(p),
+            BindAddress::Uds(uds_path) => {
+                _ = std::fs::remove_file(&uds_path);
```

Comment:
Maybe add a comment explaining why it is ok to ignore the result here (it would fail at bind time).